sleap/io/visuals.py

```diff
@@ -264,21 +268,20 @@ def _mark_single_frame(self, video_frame: np.ndarray, frame_idx: int) -> np.ndar
         Returns:
             ndarray of frame image with visual annotations added.
         """
-
         # Use OpenCV to convert to BGR color image
         video_frame = img_to_cv(video_frame)
 
         # Add the instances to the image
         overlay = self._plot_instances_cv(video_frame.copy(), frame_idx)
 
-        return cv2.addWeighted(overlay, self.alpha, video_frame, 1 - self.alpha, 0)
+        return overlay
```

Comment:
Why are we removing this?

This is necessary for overlaying the tracking with transparency on the original image. Without it, the markers and lines are at full opacity, so it's not as clear what's underneath them.

Reply:
Added this back to get some transparency